recipes/koalas/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "koalas" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9ed16834e255924ddfe9a254cbb258b35dd167b727f223bee316ee436778794d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [py<35]
+
+requirements:
+  host:
+    - numpy >=1.14
+    - pandas >=0.23
+    - pip
+    - pyarrow >=0.10
+    - python
+  run:
+    - numpy >=1.14
+    - pandas >=0.23
+    - pyarrow >=0.10
+    - python
+
+test:
+  requires:
+    - pyspark >=2.4
+  imports:
+    - databricks
+    - databricks.koalas
+    - databricks.koalas.missing
+
+about:
+  home: https://github.com/databricks/koalas
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: pandas API on Apache Spark
+  doc_url: https://koalas.readthedocs.io/
+  dev_url: https://github.com/databricks/koalas
+
+extra:
+  recipe-maintainers:
+    - rxin
```

Comment:
@rxin @gatorsmile @HyukjinKwon Are you happy to be listed as a maintainer of this recipe?

Reply:
More than happy :D

Reply:
Yes. Thanks!